src/coreclr/nativeaot/Runtime/regdisplay.h

```diff
@@ -30,7 +30,12 @@ struct REGDISPLAY
 #endif // TARGET_AMD64
 
     uintptr_t   SP;
-    PCODE        IP;
+    PCODE       IP;
+
+#ifdef TARGET_AMD64
```

Comment:
Do we want to make this new member Windows only? The ifdefs around places that work with this are somehow inconsistent - some of them check for windows too and some don't.

Reply:
I tried to make it less platform-dependent initially, but as we moved through approaches it ended up being strictly wintel. Every piece working with this field is windows-only.  
I will make the field windows-only as well.